README.md

```diff
@@ -96,13 +95,12 @@ COMMANDS
   deploy   Create a new version of your documentation from the given file or URL.
   diff     Get a comparison diff with your documentation from the given file or URL.
   help     Display help for bump.
-  overlay  Apply an OpenAPI specified overlay to your API definition.
```

Comment:
Why remove the overlay command from the current help output?

Reply:
Copy pasta. The two documents have diverged so much that copying from docs to cli has confused many things. I'll try and merge and copy the final version backwards.

Reply:
:spaghetti: thanks for the tidying up :pray: